SCClassLibrary/Common/Collections/Event.sc

```diff
@@ -44,17 +47,9 @@ Event : Environment {
 		//		^this.delta
 	}
 
-	// this[\isRest] may be nil
 	isRest {
-		^this[\isRest] == true
-		or: { this[\type] == \rest
-			or: {
-				this.use {
-					parent ?? { parent = defaultParentEvent };
-					~detunedFreq.value.isRest
-				}
-			}
-		}
+		_Event_IsRest
+		^false
```

Comment:
Both are extensible... IMO better to encourage modifications to the primitive than encourage hybrid C++/SC functions, again because it's faster and cleaner.